tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml

```diff
@@ -146,6 +146,7 @@
             <class name="org.ballerinalang.test.statements.transaction.NestedTransactionTest"/>
             <class name="org.ballerinalang.test.statements.transaction.NestedRetryTransactionStmtsTest" />
             <class name="org.ballerinalang.test.statements.transaction.SetRollbackOnlyTest" />
+            <class name="org.ballerinalang.test.expressions.object.ObjectConstructorTest" />
```

Comment:
We can add the package under `<packages>`. Then we don't have to update this xml for every new test class..